storage/mongodb/mongodb-test.yaml

```diff
@@ -0,0 +1,133 @@
+        shell: oc get project {{ test_project_name }}
+        register: get_project_result
+        until: get_project_result.stderr.find(" not found") != -1
+        retries: 11
+        delay: 10
+        ignore_errors: True
+      - name: fail the test if the project is still there after 120s
+        fail:
+          msg: "the pod is NOT deleted after 120s"
+        when: get_project_result.rc == 0
+    when: delete_test_project_before_test|bool == true
+    tags:
+      - setup
+
+  - name: create test project
+    command:  oc new-project {{ test_project_name }} --skip-config-write=true
+    tags:
+      - setup
+
+  - name: create mongodb
+    shell: oc process -f {{ tmp_folder }}/files/oc/mongodb-persistent-template.yaml -p MEMORY_LIMIT={{ MEMORY_LIMIT }} -p MONGODB_USER={{ MONGODB_USER }} -p MONGODB_PASSWORD={{ MONGODB_PASSWORD }} -p MONGODB_DATABASE={{ MONGODB_DATABASE }} -p VOLUME_CAPACITY={{ VOLUME_CAPACITY }} -p MONGODB_VERSION={{ MONGODB_VERSION }} -p STORAGE_CLASS_NAME={{ STORAGE_CLASS_NAME }} | oc create --namespace={{ test_project_name }} -f -
+    tags:
+      - setup
+
+  - block:
+      - name: wait until the mongodb pod is running
+        shell: oc get pod -n "{{ test_project_name }}" | grep -v deploy | grep mongodb | grep Running
+        register: pod_ready_result
+        until: pod_ready_result.rc == 0
+        retries: 11
+        delay: 10
+        ignore_errors: True
+      - name: fail the test if the pod is NOT running after 120s
+        fail:
+          msg: "the pod is NOT running after 120s"
+        when: pod_ready_result|failed
+    tags:
+      - setup
+
+  - name: create ycsb
+    shell: oc create  --namespace={{ test_project_name }} -f {{ tmp_folder }}/files/oc/dc_ycsb.yaml
+    tags:
+      - setup
+
+  - block:
+      - name: wait until the ycsb pod is running
+        shell: oc get pod -n "{{ test_project_name }}" | grep -v deploy | grep ycsb | grep Running
+        register: pod_ready_result
+        until: pod_ready_result.rc == 0
+        retries: 11
+        delay: 10
+        ignore_errors: True
+      - name: fail the test if the pod is NOT running after 120s
+        fail:
+          msg: "the pod is NOT running after 120s"
+        when: pod_ready_result|failed
+    tags:
+      - setup
+
+  - name: register pbench
+    shell: |
+      pbench-kill-tools
+      pbench-clear-tools
+      pbench-clear-results
+      pbench-register-tool-set
+      pbench-register-tool --name=oc
+
+      NODES=($(oc get node  --no-headers | grep -v master | awk '{print $1}'))
+
+      for node in "${NODES[@]}"
+      do
+        pbench-register-tool-set --remote="${node}"
+      done
+    when: pbench_registration|bool == true
+    tags:
+      - run
+
+  - name: run the benchmarks
+    command: pbench-user-benchmark --config="mongodb_storage_test_{{ STORAGE_CLASS_NAME }}" -- "{{ tmp_folder }}/files/scripts/test-mongo.sh" "{{ test_project_name }}" "{{ iteration }}" "{{ ycsb_threads }}" "{{ workload }}"
```

Comment:
if we can change this 
```
--config="mongodb_storage_test_{{ STORAGE_CLASS_NAME }}"
``` 
to 
```
--config="mongodb_storage_test_{{ STORAGE_CLASS_NAME }}_{{ workload }}"
``` 
it will help to differ pbench results based on workload ran 
eg, directory name 
``` 
pbench-user-benchmark_mongodb_storage_test_glusterfs-storage_workloada,workloadb_2018.05.08T11.23.59
```

Reply:
Done. Also added the size of MEM as discussed.